app/controllers/concepts_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ConceptsController < ApplicationController
+  before_action :data_check, :build_request, :disable_top_navigation
+
+  ROUTE_MAP = {
+    show: proc { |params| Parliament::Utils::Helpers::ParliamentHelper.parliament_request.concept_by_id.set_url_params({ concept_id: params[:concept_id] }) }
+  }.freeze
+
+  def show
+    concepts = Parliament::Utils::Helpers::FilterHelper.filter(@request, 'Concept')
+
+    # Finds the concept we are looking for, as GET request may return multiple Concepts
+    @concept = concepts.find { |concept| concept.graph_id == params[:concept_id] }
+    raise ActionController::RoutingError, 'Concept Not Found' unless @concept
+
+    # Returns array of Concept Grom::Nodes
+    @parent_concepts = @concept.parent_concepts.sort_by(&:name)
```

Comment:
There are no _parent_ concepts and _child_ concepts.
There are only **broader** and **narrower** ones.
I highly recommend using established nomenclature throughout to avoid confusion.